runtime/doc/terminal.txt

```diff
@@ -411,6 +412,11 @@ to point to the right file, if needed.  If you have both the 32-bit and 64-bit
 version, rename to winpty32.dll and winpty64.dll to match the way Vim was
 build.
 
+ConPTY is now available. It is a technology supported by a large amount of
```

Comment:
I might suggest the following text:

```
On more recent versions of MS-Windows (beginning with the "October 2018
Update"), winpty is no longer required. On those versions, |:terminal| will use
Windows' built-in support for hosting terminal applications, "ConPTY".  When
ConPTY is in use, there may be rendering artifacts regarding ambiguous-width
characters. If you encounter any such issues, set 'termmode' to winpty.
```

This will also add documentation links to `termmode` and `:terminal`.

Reply:
That's excellent! Replace it.